backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -75,11 +76,23 @@ object BpkTheme {
 
   val typography: BpkTypography
     @Composable
-    get() = LocalBpkTypography.current
+    get() = if (LocalInspectionMode.current) {
+      // when in preview mode return a default typography object to ensure previews work
+      // without wrapping it in another composable
+      BpkTypography(defaultFontFamily = FontFamily.SansSerif)
```

Comment:
I like this approach, just thinking about possible performance issues we may have here. Theoretically, there shouldn't be any and we're good to go.

Reply:
re performance issues: yeah the only thing this would mean that if `LocalInspectionMode` would change everything would be recomposed, but this should never be the case

and yes it won't be 100% accurate, but it should be good enough for our use case - in the app previews will more likely be separate preview composables that can take the theme wrapper as well than the use case we have here where the preview is used for tests & the demo app